container_pipeline/workers/base.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+import os
+
+from container_pipeline.lib import settings
+from container_pipeline.lib.openshift import Openshift
+from container_pipeline.lib.queue import JobQueue
+
+
+class BaseWorker(object):
+    """Base class for pipeline workers"""
+
+    def __init__(self, logger=None):
+        self.queue = JobQueue(host=settings.BEANSTALKD_HOST,
+                              port=settings.BEANSTALKD_PORT)
+        self.logger = logger or logging.getLogger('console')
+        self.openshift = Openshift(logger=self.logger)
+        self.openshift.login()
+
+    def handle_job(self, job):
+        """Handle job"""
+        raise NotImplementedError
+
+    def notify_build_failure(self, namespace, notify_email, build_logs_file,
+                             project, jobid, test_tag):
+        """Notify build failure"""
+        data = {}
+        data['action'] = 'notify_user'
+        data['namespace'] = namespace
+        data['build_status'] = False
+        data['notify_email'] = notify_email
+        data['build_logs_file'] = build_logs_file
+        data['project_name'] = project
+        data['job_name'] = jobid
```

Comment:
Can we try and use same key name and value variable on left and right? For eg:
```python
data['project_name'] = project_name
data['job_name'] = job_name
```
OR
```python
data['project'] = project
data['jobid'] = jobid
```
instead of current implementation? This is to avoid confusions. We've made mistakes in past owing to such implementations.